lib/galaxy/util/__init__.py

```diff
@@ -848,13 +847,18 @@ def unicodify( value, encoding=DEFAULT_ENCODING, error='replace', default=None )
     """
     Returns a unicode string or None
     """
-
-    if isinstance( value, text_type ):
-        return value
+    if value is None:
+        return None
     try:
-        return text_type( str( value ), encoding, error )
-    except:
+        if not isinstance(value, string_types):
```

Comment:
Don't forget to update `unicodify()` to this version: https://gist.github.com/nsoranzo/6ab474f2ae74c04b0c0c